metatron-app/metatron-discovery
# fix error black theme style

### Description
1. The thumbnail icon of the empty dashboard is changed depending on the selection.
2. Design tooltips and window tooltips pop up at the same time in items in the workspace
3. The data source list layer in the dashboard list does not display correctly.
4. Black theme Legend text is not visible.
5. The explosion title font is incorrect.

**Related Issue** : 



### How Has This Been Tested?
1. Make sure that the Thumbnail icon on the empty dashboard displays well regardless of the selection.
2. Verify that the window tooltips do not appear when the design tooltips are displayed in items in the workspace.
3. Verify that the data source list layer in the dashboard list is displayed normally.
4. Make sure the legend text is visible at the time of the black theme.
5. Check if the explosion title font is normal.

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
